@utahdts/utah-design-system/react/components/containers/drawer/DrawerTitle.jsx

```diff
@@ -0,0 +1,26 @@
+import { joinClassNames } from '../../../util/joinClassNames';
+
+/**
+ * @param {object} props
+ * @param {import('react').ReactNode} props.children
+ * @param {string} [props.className]
+ * @param {string} props.id
```

Comment:
does `id` need to be required?

Reply:
Put a comment on the jsdoc to explain its use?